logstash-core/lib/logstash/agent.rb

```diff
@@ -23,19 +29,28 @@ class LogStash::Agent
   def initialize(params)
     @logger = params[:logger]
     @auto_reload = params[:auto_reload]
-    @pipelines = {}
+    @debug  = params.fetch(:debug, false)
 
+    @pipelines = {}
+    @started_at = Time.now
```

Comment:
can I suggest we move this into a constant under LogStash::STARTED_AT that is evaluated in the runner or here in the agent?